src/components/WorkoutBlock.tsx

```diff
@@ -32,23 +47,30 @@ const WorkoutBlock: React.FC<WorkoutBlockProps> = ({ editing, workout, updateNam
   }
 
   return (
-    <BlockContainer>
-      <TouchableOpacity
-        accessibilityRole="button"
-        className="flex h-full flex-1 flex-row items-center p-1"
-        onPress={() => onPress(workout)}
-        hitSlop={{ top: 20, bottom: 20, left: 20 }}
-      >
-        <Text className="text-base font-medium">{name}</Text>
-      </TouchableOpacity>
-      <TouchableOpacity
-        accessibilityRole="button"
-        className="flex h-full flex-row items-center justify-center p-1"
-        hitSlop={{ top: 20, bottom: 20, right: 20 }}
-      >
-        <Ionicon name="ellipsis-horizontal" size={16} />
-      </TouchableOpacity>
-    </BlockContainer>
+    <View>
```

Comment:
You could use empty angle brackets `<> </>` instead of a `View` here. The problem with using a `View` component is that it can interfere with styling and style cascading.